js/currencycom.js

```diff
@@ -948,15 +863,15 @@ module.exports = class currencycom extends Exchange {
         //
         //     [
         //         {
-        //             "a":1658318071,  // Aggregate tradeId
-        //             "p":"0.02476",   // Price
-        //             "q":"0.0115",    // Official doc says: "Quantity (should be ignored)"
+        //             "a":1658318071,
+        //             "p":"0.02476",
+        //             "q":"0.0",
         //             "T":1591001423382,
-        //             "m":false        // Was the buyer the maker
+        //             "m":false
         //         }
         //     ]
         //
-        return this.parseTrades (response, market, since, limit, params);
+        return this.parseTrades (response, market, since, limit);
```

Comment:
isn't it better to include 'params' too ?

Reply:
Not in this case – these params are for extending the resulting trades, so we don't want to extend all trades with any possible params a user might pass to this unified method.